modules/hooks/item.js

```diff
@@ -53,18 +53,17 @@ export default function () {
       // If critical, subtract wounds value from actor's
       if (item.type == "critical") {
         let newWounds;
-        if (Number.isNumeric(item.wounds.value))
-        {
-          if (item.wounds.value.toLowerCase() == "death")
+        let appliedWounds = Number.parseInt(item.wounds.value);
+        if (Number.isInteger(appliedWounds)) {
+          newWounds = item.actor.status.wounds.value - appliedWounds;
+          if (newWounds < 0) {
             newWounds = 0;
-
-            newWounds = item.actor.status.wounds.value - Number(item.wounds.value)
-            if (newWounds < 0) newWounds = 0;
-            
-            item.actor.update({ "system.status.wounds.value": newWounds });
-            ui.notifications.notify(`${item.wounds.value} ${game.i18n.localize("CHAT.CriticalWoundsApplied")} ${item.actor.name}`)
+          }
+        } else if (item.wounds.value.toLowerCase() == "death") {
+          newWounds = 0;
         }
-
+        item.actor.update({ "system.status.wounds.value": newWounds });
+        ui.notifications.notify(`${item.wounds.value} ${game.i18n.localize("CHAT.CriticalWoundsApplied")} ${item.actor.name}`)
```

Comment:
What is the purpose of this change? It seems like a partial reversion of https://github.com/moo-man/WFRP4e-FoundryVTT/commit/6934611265d87ce8103430085f1ab4c18b52374c

I don't see any mention of what this fixes in the previous PR either

Reply:
```javascript
        if (Number.isNumeric(item.wounds.value))
        {
          if (item.wounds.value.toLowerCase() == "death")
            newWounds = 0;
            newWounds = item.actor.status.wounds.value - Number(item.wounds.value)
            if (newWounds < 0) newWounds = 0;

            item.actor.update({ "system.status.wounds.value": newWounds });
            ui.notifications.notify(`${item.wounds.value} ${game.i18n.localize("CHAT.CriticalWoundsApplied")} ${item.actor.name}`)
        }
```

I never really understood this original piece of code. First of all, i was confused about isNumeric function that is not a standard JS (probably comes from jQuery or something extension?). Second of all, why compare to == "death" is inside of IsNumeric condition? besides that i dont have anything else to add.

Reply:
Yeah good point, it's based off some very early code, and I didn't realize `isNumeric` isn't native javascript

My main concern is just making sure it handles UiA's trivial critical hits (why they didn't just use 0 instead of T for the wounds value i'll never understand).